gtda/images/filtrations.py

```diff
@@ -489,11 +493,10 @@ def transform(self, X, y=None):
 @adapt_fit_transform_docs
 class ErosionFiltration(BaseEstimator, TransformerMixin):
     """Filtrations of 2D/3D binary images based on the erosion of activated
-    regions.
+    regions. Binary erosion is a morphological operator commonly used in
```

Comment:
I thought we previously pushed sentence 2 of this paragraph to the next paragraph?